Source/objects.h

```diff
@@ -49,6 +50,36 @@ struct ObjectStruct {
 	uint32_t _oVar6;
 	_speech_id _oVar7;
 	int _oVar8;
+
+	/**
+	 * @brief Marks the map region to be refreshed when the player interacts with the object.
+	 *
+	 * Some objects will cause a map region to change when a player interacts with them (e.g. Skeleton King
+	 * antechamber levers). The coordinates used for this region are based on a 40*40 grid overlaying the central
+	 * 80*80 region of the dungeon.
+	 *
+	 * @param topLeft corner of the map region closest to the origin
+	 * @param bottomRight corner of the map region furthest from the origin
+	 * @param v ID/discriminator for the object type? Needs to be investigated further
+	 */
+	constexpr void SetMapRange(Point topLeft, Point bottomRight, int v)

```

Comment:
I'd suggest suffixing the arguments with "position" for consistency and semantic value. So `topLeftPosition` and `bottomRightPosition`.